src/primitives.rs

```diff
@@ -32,5 +32,18 @@ sizeof_impl!(
     AtomicU8, AtomicU16, AtomicU32, AtomicU64, AtomicUsize,
     AtomicI8, AtomicI16, AtomicI32, AtomicI64, AtomicIsize,
     NonZeroU8, NonZeroU16, NonZeroU32, NonZeroU64, NonZeroU128, NonZeroUsize,
-    NonZeroI8, NonZeroI16, NonZeroI32, NonZeroI64, NonZeroI128, NonZeroIsize
+    NonZeroI8, NonZeroI16, NonZeroI32, NonZeroI64, NonZeroI128, NonZeroIsize,
+    Wrapping<u8>, Wrapping<u16>, Wrapping<u32>, Wrapping<u64>, Wrapping<u128>, Wrapping<usize>,
+    Wrapping<i8>, Wrapping<i16>, Wrapping<i32>, Wrapping<i64>, Wrapping<i128>, Wrapping<isize>
 );
+
+#[cfg(feature = "saturating_impls")]
```

Comment:
Can the macro call be feature gated, instead of needing the wrapping module?

Reply:
I also want to put the `use core::num::Saturating` inside the mod just so I don't have to repeat the full path name, but I can put two `#[cfg]`s, one on the `use` and another on the macro call